spec/ldoc.css

```diff
@@ -1,291 +1,370 @@
 }
 
-a:link { color: #008fee; }
-a:visited { color: #008fee; }
-a:hover { color: #22a7ff; }
+body {
+  color: #47555c;
+  font-size: 16rem;
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  background: #eff4ff;
+}
 
-h1 { font-size:26px; font-weight: normal; }
-h2 { font-size:22px; font-weight: normal; }
-h3 { font-size:18px; font-weight: normal; }
-h4 { font-size:16px; font-weight: bold; }
+a:link,
+a:visited {
+  color: #008fee;
+}
+a:hover,
+a:active {
+  color: #22a7ff;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  font-weight: normal;
+}
+
+h1 {
+  font-size: 26rem;
+}
+h2 {
+  font-size: 22rem;
+}
+h3 {
+  font-size: 18rem;
+}
+h4 {
+  font-size: 16rem;
+}
 
 hr {
-    height: 1px;
-    background: #c1cce4;
-    border: 0px;
-    margin: 15px 0;
+  height: 1rem;
+  background: #c1cce4;
+  border: 0rem;
+  margin: 15rem 0;
 }
 
-code, tt {
-    font-family: monospace;
+code,
+tt {
+  font-family: monospace;
 }
 span.parameter {
-    font-family: monospace;
-    font-weight: bold;
-    color: rgb(99, 115, 131);
+  font-family: monospace;
+  font-weight: bold;
+  color: rgb(99, 115, 131);
 }
-span.parameter:after {
-    content:":";
+span.parameter::after {
+  content: ":";
 }
-span.types:before {
-    content:"(";
+span.types::before {
+  content: "(";
 }
-span.types:after {
-    content:")";
+span.types::after {
+  content: ")";
 }
 .type {
-    font-weight: bold; font-style:italic
+  font-weight: bold;
+  font-style: italic;
 }
 
 p.name {
-    font-family: "Andale Mono", monospace;
+  font-family: "Andale Mono", monospace;
 }
 
 #navigation {
-    float: left;
-    background-color: white;
-    border-right: 1px solid #d3dbec;
-    border-bottom: 1px solid #d3dbec;
+  float: left;
+  background-color: white;
+  border-right: 1rem solid #d3dbec;
+  border-bottom: 1rem solid #d3dbec;
 
-    width: 14em;
-    vertical-align: top;
-    overflow: visible;
+  width: 14em;
+  vertical-align: top;
+  overflow: visible;
 }
 
 #navigation br {
-    display: none;
+  display: none;
 }
 
 #navigation h1 {
-    background-color: white;
-    border-bottom: 1px solid #d3dbec;
-    padding: 15px;
-    margin-top: 0px;
-    margin-bottom: 0px;
+  background-color: white;
+  border-bottom: 1rem solid #d3dbec;
+  padding: 15rem;
+  margin-top: 0rem;
+  margin-bottom: 0rem;
 }
 
 #navigation h2 {
-    font-size: 18px;
-    background-color: white;
-    border-bottom: 1px solid #d3dbec;
-    padding-left: 15px;
-    padding-right: 15px;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    margin-top: 30px;
-    margin-bottom: 0px;
+  font-size: 18rem;
+  background-color: white;
+  border-bottom: 1rem solid #d3dbec;
+  padding-left: 15rem;
+  padding-right: 15rem;
+  padding-top: 10rem;
+  padding-bottom: 10rem;
+  margin-top: 30rem;
+  margin-bottom: 0rem;
 }
 
 #content h1 {
-    background-color: #2c3e67;
-    color: white;
-    padding: 15px;
-    margin: 0px;
+  background-color: #2c3e67;
+  color: white;
+  padding: 15rem;
+  margin: 0rem;
 }
```

Comment:
Replace `px` with `rem` directly won't work.

> rem Represents the [font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size) of the root element (typically [<html>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html)). When used within the root element [font-size](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size), it represents its initial value (a common browser default is 16px, but user-defined preferences may modify this).

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/length

Reply:
Thats why i altered the default browser font-size in html
`html{
font-size:62.5%;
}`
62.5% i.e. 10px as the default